Zend/tests/try/catch_novar_2.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+catch without capturing a variable - exception in destructor
+--FILE--
+<?php
+class ThrowsOnDestruct extends Exception {
+    public function __destruct() {
+        echo "Throwing\n";
+        throw new RuntimeException(__METHOD__);
+    }
+}
+try {
+    throw new ThrowsOnDestruct();
+} catch (Exception $ex) {
```

Comment:
Shouldn't `$ex` be removed here?

Reply:
I must've reinstated it during my experiments. Fixed.